spring-kafka/src/main/java/org/springframework/kafka/listener/AbstractConsumerSeekAware.java

```diff
@@ -82,20 +86,20 @@ public void unregisterSeekCallback() {
 	}
 
 	/**
-	 * Return the callback for the specified topic/partition.
+	 * Return the callbacks for the specified topic/partition.
 	 * @param topicPartition the topic/partition.
-	 * @return the callback (or null if there is no assignment).
+	 * @return the callbacks (or null if there is no assignment).
 	 */
 	@Nullable
-	protected ConsumerSeekCallback getSeekCallbackFor(TopicPartition topicPartition) {
+	protected List<ConsumerSeekCallback> getSeekCallbacksFor(TopicPartition topicPartition) {
```

Comment:
This is a breaking change. Since there is no easy path for us to deprecate first and then remove later in this case, I suggest calling this in the `whats-new` section.

Reply:
I think it is better to leave an old method yet, even if it is deprecated.
Probably could just return a first item from the list.
This way no breaking changes and the is a migration path.
The name of method are definitely different, therefore no way we would break end-user code.

Reply:
Thanks for your great suggestion!

I'll make the changes and add new method (`getSeekCallbacksFor()`) in the `whats-new` section as you mentioned.